pkg/tracker/index.go

```diff
@@ -182,25 +190,36 @@ func (j *JSONfile) Get() ([]byte, error) {
 }
 
 func (i *IndexTracker) Exec(ctx context.Context) error {
-	payload, err := i.Source.Get()
-	if err != nil {
-		return err
+	now := clck.Now()
+	currentVal := apiOracle.GetLatest(i.Identifier)
+	runID := true
+	if currentVal != nil {
+		if uint(now.Sub(currentVal.Created)) < i.Interval {
+			runID = false
```

Comment:
```suggestion
			return nil
```
Instead of this, you can just return here.

Btw I think a more simple approach could be to just add a variable `lastRunTimestamp`

This way you can have something like
```
if now.Sub(i.lastRunTimestamp)< i.Interval{
	return nil
}
i.lastRunTimestamp = now

```

Reply:
What do you think about this suggestion? I find it a bit easier to understand